lib/hook.js

```diff
@@ -46,6 +46,22 @@ function Hook() {
 	};
 }
 
+function findInDir (dir, include = /\.js$/, exclude = /^[._]/, fileList = []) {
```

Comment:
<!--
  0 failure: 
  3 warning:  lib/hook.js line ..., lib/hook.js line ..., lib/hook.js line ...
  
  
  DangerID: danger-id-Danger;
  File: lib/hook.js;
  Line: 49;
-->

- :warning: lib/hook.js line 49 – Default parameters are not supported until Node.js 6.0.0. The configured version range is '>=4.0'. (node/no-unsupported-features/es-syntax)
- :warning: lib/hook.js line 49 – Default parameters are not supported until Node.js 6.0.0. The configured version range is '>=4.0'. (node/no-unsupported-features/es-syntax)
- :warning: lib/hook.js line 49 – Default parameters are not supported until Node.js 6.0.0. The configured version range is '>=4.0'. (node/no-unsupported-features/es-syntax)

Reply:
As the warning implies, we need to bump the Node range to support default parameters and that would require a major rev bump. I don't see any need to do that when you can just do it the old way for now.

Reply:
Also, no space between the function name and the parenthesis. Not sure why eslint isn't catching this stuff.

Reply:
I bumped the min node version to 8.0